question/bank/export/export.php

```diff
@@ -54,7 +55,7 @@
 
 if ($from_form = $export_form->get_data()) {
     $thiscontext = $contexts->lowest();
-    if (!is_readable("format/{$from_form->format}/format.php")) {
+    if (!is_readable($CFG->dirroot . "/question/format/{$from_form->format}/format.php")) {
         print_error('unknowformat', '', '', $from_form->format);
```

Comment:
i think print_error is deprecated, recommendation would be to use moodle_exception instead